backend/utils/passport-setup.js

```diff
@@ -1,14 +1,19 @@
 const GoogleStrategy = require("passport-google-oauth20").Strategy;
 const passport = require("passport");
-const router = require("express").Router();
+const User = require("../models/User");
 
 passport.serializeUser((user, done) => {
   done(null, user);
```

Comment:
I think, but I'm not totally sure, we just can store the id here and thus just return a unique id (you can decide if that's best being the `_id` field or `oauthId`. 

[This](https://stackoverflow.com/a/27637668) might help.